src/current/_includes/releases/v24.3/v24.3.0-alpha.2.md

```diff
@@ -0,0 +1,88 @@
+## v24.3.0-alpha.2
+
+Release Date: October 14, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-3-0-alpha-2-security-updates">Security updates</h3>
+
+- The parameters for an [HBA config entry]({% link v24.3/security-reference/authentication.md %}#hba-configuration-syntax) for LDAP are now validated when the entry is created or amended, in addition to the validation that happens during an authentication attempt. [#132086][#132086]
+
+- Added automatic cleanup and validation for dropped roles when [altering default privileges]({% link v24.3/alter-default-privileges.md %}). [#131782][#131782]
```

Comment:
@annrpom PTAL

Reply:
@mdlinville 
```suggestion
- Added automatic cleanup and validation for for [default privileges]({link}) that reference dropped roles
```
And then the right link to https://www.cockroachlabs.com/docs/v24.2/security-reference/authorization#default-privileges instead

The current note sounds like the cleanup/validation happens during an `ALTER DEFAULT PRIVILEGE` statement, but it happens during version upgrade/finalization. Otherwise, LGTM!

Reply:
Thanks, I edited your edit before applying it.